src/shared/components/query/GeneSymbolValidator.tsx

```diff
@@ -39,6 +39,16 @@ export default class GeneSymbolValidator extends QueryStoreComponent<{}, {}>
 		if (!this.store.oql.query.length)
 			return null;
 
+		if (this.store.isQueryLimitReached)
+			return (
+				<div className={styles.GeneSymbolValidator}>
+					<div className={styles.invalidBubble} title="Please limit your queries to 100 genes or fewer.">
+						<FontAwesome className={styles.icon} name='exclamation-circle'/>
+						<span>Please query no more than 100 genes.</span>
```

Comment:
@dippindots lets add the language about contact us if use case requires more ...

Reply:
lets just make it read exactly the same: 
"Queries are limited to 100 genes. Please <a href={`mailto:${AppConfig.serverConfig.skin_email_contact}`}>let us know</a> your use case(s) if you need to query more than 100 genes."